docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/keeping-cohesion-with-unified/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Attribution"
```

Comment:
This page is nested under Our Packages > Attribution > Attribution. Should it still be inside the main attribution page?